packs/data/fall-of-plaguestone.db/spear-launcher.json

```diff
@@ -0,0 +1,165 @@
+{
+    "_id": "jpw5eHse4EcmYSp6",
+    "data": {
+        "attributes": {
+            "ac": {
+                "value": 18
+            },
+            "hardness": 8,
+            "hp": {
+                "details": "",
+                "max": 32,
+                "temp": 0,
+                "value": 32
+            },
+            "stealth": {
+                "details": "<p>(trained)</p>",
+                "value": 10
+            }
+        },
+        "creatureType": "",
+        "customModifiers": {},
+        "details": {
+            "description": "<p>An old heavy crossbow is hidden in a pile of trash, loaded with a wooden spear, and connected to the rope holding the door.</p>",
+            "disable": "<p>@Check[type:thievery|dc:18|traits:action:disable-device]{Thievery (trained)} on the rope allows a PC to tie the rope off and open the door without setting off the trap.</p>",
+            "isComplex": false,
+            "level": {
+                "value": 2
+            },
+            "reset": "",
+            "routine": ""
+        },
+        "saves": {
+            "fortitude": {
+                "saveDetail": "",
+                "value": 11
+            },
+            "reflex": {
+                "saveDetail": "",
+                "value": 3
+            },
+            "will": {
+                "saveDetail": "",
+                "value": 0
+            }
+        },
+        "source": {
+            "author": "",
+            "value": "Pathfinder Adventure: The Fall of Plaguestone"
+        },
+        "statusEffects": [],
+        "traits": {
+            "di": {
+                "custom": "",
+                "selected": {},
+                "value": [
+                    "critical-hits",
+                    "object-immunities",
+                    "precision"
+                ]
+            },
+            "dr": [],
+            "dv": [],
+            "rarity": "common",
+            "size": {
+                "value": "med"
+            },
+            "traits": {
+                "custom": "",
+                "value": [
+                    "mechanical",
+                    "trap"
+                ]
+            }
+        }
+    },
+    "img": "systems/pf2e/icons/default-icons/hazard.svg",
+    "items": [
+        {
+            "_id": "YapaAOg2pzlW9uPP",
+            "data": {
+                "actionCategory": {
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "reaction"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p><strong>Trigger</strong> The rope is cut or otherwise untied from the door.</p>\n<hr />\n<p><strong>Effect</strong> The trap makes an attack against the creature that manipulated the rope.</p>"
```

Comment:
```suggestion
                    "value": "<p><strong>Trigger</strong> The rope is cut or otherwise untied from the door</p>\n<hr />\n<p><strong>Effect</strong> The trap makes an attack against the creature that manipulated the rope</p>"
```